docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -0,0 +1,78 @@
+# Upgrade to 8.0.0
```

Comment:
I think there should be information in this document saying that the default configuration file has changed.

Reply:
A new section has been added.